rbac/models.py

```
@@ -0,0 +1,1009 @@
+        verbose_name='زمان به‌روزرسانی'
+    )
+    
+    last_activity = models.DateTimeField(
+        null=True, 
+        blank=True,
+        verbose_name='آخرین فعالیت'
+    )
+    
+    # تنظیمات احراز هویت
+    USERNAME_FIELD = 'phone_number'
+    REQUIRED_FIELDS = ['first_name', 'last_name']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)